frontend/main.js

```diff
@@ -534,12 +534,17 @@ async function startApp () {
         await syncContractsInOrder(identityContractID, contractIDs)
 
         if (this.ephemeral.finishedLogin === 'yes') return
-        return sbp('gi.app/identity/login', { identityContractID }).catch((e) => {
-          console.error(`[main] caught ${e?.name} while logging in: ${e?.message || e}`, e)
-          console.warn(`It looks like the local user '${identityContractID}' does not exist anymore on the server 😱 If this is unexpected, contact us at https://gitter.im/okTurtles/group-income`)
-        })
+        return sbp('gi.app/identity/login', { identityContractID })
       }).catch(e => {
         console.error(`[main] caught ${e?.name} while fetching settings or handling a login error: ${e?.message || e}`, e)
+
+        sbp('gi.ui/prompt', {
+          heading: L('Failed to login'),
+          question: e.message
+            ? L('Error details:{br_}{err}', { err: e.message, ...LTags() })
+            : L('Error details: {reportError}', ...LError(e)),
```

Comment:
Hmm, this is a bit confusing, because it calls `LError` if `e.message` is not defined, but `LError` is written in a way that expects `e.message` to be defined:

```js
  return {
    reportError: L('"{errorMsg}". You can {a_}report the error{_a}.', {
      errorMsg: error.message,
      'a_': `<a class="link" target="_blank" href="${url}">`,
      '_a': '</a>'
    })
  }
```